src/App.tsx

```diff
@@ -19,93 +16,83 @@ import MapPage from './components/Maps/MapPage';
 
 function App() {
   return (
-    <ThemeProvider theme={Theme}>
-      <SnackbarProvider
-        anchorOrigin={{ horizontal: 'center', vertical: 'bottom' }}
-      >
-        <Switch>
-          <Route exact path="/events" component={EventsPage} />
-          <Route
-            exact
-            path="/events/new"
-            component={(props) => <EventCreationPage mode="new" {...props} />}
-          />
-          <Route
-            exact
-            path="/events/:eventId/ccps/new"
-            component={(props) => <CCPFormPage mode="new" {...props} />}
-          />
-          <Route
-            exact
-            path="/events/:eventId/ccps/:ccpId/edit"
-            component={(props) => <CCPFormPage mode="edit" {...props} />}
-          />
-          <Route exact path="/events/:eventId" component={EventDashboardPage} />
-          <Route
-            exact
-            path="/events/:eventId/edit"
-            component={(props) => <EventCreationPage mode="edit" {...props} />}
-          />
-          <Route
-            exact
-            path="/manage/:resource"
-            component={ResourceOverviewPage}
-          />
-          <Route exact path="/manage/">
-            <Redirect to="/manage/members" />
-          </Route>
-          <Route
-            exact
-            path="/manage/hospitals/:mode/:hospitalId?"
-            component={HospitalFormPage}
-          />
-          <Route
-            exact
-            path="/manage/ambulances/:mode/:ambulanceId?"
-            component={AmbulanceFormPage}
-          />
-          <Route
-            exact
-            path="/manage/members/:mode/:userId?"
-            component={UserFormPage}
-          />
-          <Route
-            exact
-            path="/events/:eventId/ccps/:ccpId/scan"
-            component={ScanPatientPage}
-          />
-          <Route
-            exact
-            path="/events/:eventId/ccps/:ccpId/manual"
-            component={EnterBarcodePage}
-          />
-          <Route
-            exact
-            path="/events/:eventId/ccps/:ccpId"
-            component={CCPDashboardPage}
-          />
-          <Route
-            exact
-            path="/events/:eventId/ccps/:ccpId/open/:patientId"
-            component={(props) => <CCPDashboardPage {...props} />}
-          />
-          <Route
-            exact
-            path="/events/:eventId/ccps/:ccpId/patients/new/:barcodeValue?"
-            component={(props) => <PatientProfilePage mode="new" {...props} />}
-          />
-          <Route
-            exact
-            path="/events/:eventId/ccps/:ccpId/patients/:patientId"
-            component={(props) => <PatientProfilePage mode="edit" {...props} />}
-          />
-          <Route exact path="/events/:eventId/map" component={MapPage} />
-          <Route path="/">
-            <Redirect to="/events" />
-          </Route>
-        </Switch>
-      </SnackbarProvider>
-    </ThemeProvider>
+    <Switch>
```

Comment:
Basically just moving `ThemeProvider` and `SnackbarProvider` out to the parent component